projects/demo/src/modules/components/wysiwyg/examples/import/insert-template.txt

```diff
@@ -0,0 +1,5 @@
+<tui-editor
+    exampleText="Example"
+    [(ngModel)]="model">
```

Comment:
Codestyle 😨

Reply:
I replaced editor with wysiwyg now, but what wrong with codestyle?

Reply:
`>` is not on a new line 
![image](https://user-images.githubusercontent.com/11832552/132520606-fd5b2d8f-cfb6-4c34-bca6-ef202b9b4f11.png)